bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -232,7 +251,7 @@ public Response startDialog(
             @QueryParam("ksId") @Parameter(description = "keywork spotter ID") @Nullable String ksId,
             @QueryParam("sttId") @Parameter(description = "Speech-to-Text ID") @Nullable String sttId,
             @QueryParam("ttsId") @Parameter(description = "Text-to-Speech ID") @Nullable String ttsId,
-            @QueryParam("hliId") @Parameter(description = "interpreter ID") @Nullable String hliId,
+            @QueryParam("hliIds") @Parameter(description = "comma separated list of interpreter IDs") @Nullable String hliIds,
```

Comment:
This is not internal. Please keep the old `hliId`, mark it as deprecated and properly handle it in code. The new `hliIds` should return a `List<String>`, not a `String`.

Reply:
The String type was a way to keep compatibility.
But I agree with you with th equery param which must not be renamed.